.github/pull_request_template.md

```diff
@@ -12,10 +12,10 @@ Please delete options that are not relevant.
 
 # Checklist:
 
-- [ ] My content uses the appropriate readme template
+- [ ] My content uses the appropriate readme [template](https://github.com/Azure/azure-sphere-gallery/tree/main/Templates)
```

Comment:
Can we add a line to the effect that my content contains a single README.md file that is based on the template(s) and is named README.md?

I know it sounds stupid, but you'd be surprised how many people don't rename a template -- or clean up the contents--when they use one.